### ISSUE TITLE
Deno doc output missing

### ISSUE THREAD

**Issue by davidcallanan** [denoland/deno#6816]
I'm not sure if this is intentional, but much of the deno doc output is missing when using the default output format (not json).

For example, only the names of interfaces are displayed and not their functions:

![image](https://user-images.githubusercontent.com/40340425/87925660-86857180-ca78-11ea-9190-fce7e0086d38.png)

Deno version: `1.2.0`, Windows 10

**Comment by lucacasonato** [comment]
To get details about a specific symbol you should pass it to deno doc after the file name. Example: `deno doc ./mod.ts StandaloneHost`. The help text for deno doc reads:
```
~ ❯❯❯ deno help doc
deno-doc
Show documentation for a module.

Output documentation to standard output:
    deno doc ./path/to/module.ts

Output documentation in JSON format:
    deno doc --json ./path/to/module.ts

Target a specific symbol:
    deno doc ./path/to/module.ts MyClass.someField

Show documentation for runtime built-ins:
    deno doc
    deno doc --builtin Deno.Listener

USAGE:
    deno doc [OPTIONS] [ARGS]

OPTIONS:
    -h, --help                        Prints help information
        --json                        Output documentation in JSON format.
    -L, --log-level <log-level>       Set log level [possible values: debug, info]
    -q, --quiet                       Suppress diagnostic output
    -r, --reload=<CACHE_BLOCKLIST>    Reload source code cache (recompile TypeScript)
        --unstable                    Enable unstable APIs

ARGS:
    <source_file>
    <filter>         Dot separated path to symbol.
```

**Comment by SyrupThinker** [comment]
That is intentional to reduce the amount of information printed in the general case.

You can add for example `StandaloneHost` to the invocation to get that specific interface with detailed information.

**Comment by davidcallanan** [comment]
@lucacasonato @SyrupThinker Thanks, that's great! Is there any method to show detailed information for everything?

**Comment by lucacasonato** [comment]
Not at the moment. Would be a good addition though IMO. Maybe --full

**Comment by sebastienfilion** [comment]
@davidcallanan When you say everything do you mean the whole project? What if we implemented it as a blob? `deno doc ./**/*.ts`

**Comment by davidcallanan** [comment]
No, I mean all details of a *single* module i.e. a single file, usually "mod.ts".

Of course multiple files could be useful too, but often there is duplicate entries because most modules export everything in mod.ts anyway. It depends whether your module has sub-path support or whatever you want to call it. Blobs could be useful in this situation.

**Comment by humancalico** [comment]
@SyrupThinker Can you also add 'Closes #4755' in your PR?
